src/main/java/ir/sahab/walconsumer/WalConsumer.java

```diff
@@ -49,11 +49,13 @@ public class WalConsumer implements Closeable {
     private static final String NUMBER_OF_RECORDS = "_num_wal_records";
```

Comment:
```suggestion
    private static final String NUMBER_OF_RECORDS = "_num_records";
```

Also change "_wal_state" to "state".

Reply:
"state" or "_state"